views/Query/Collapsed.jsx

```diff
@@ -140,7 +140,9 @@ export default React.createClass({
                 </div>
               </div>
             </div>
-            <div className="query--date-buttons-container">
+            <div className={this.state.query.text.length ?
+                ("query--date-buttons-container") :
+                ("query--date-buttons-container query--date-buttons-disabled")}>
```

Comment:
why not pass a `disabled` property to each of the `this.state.dateButtons` ? seems like all the other date button manipulation is handled that way